index.html

```diff
@@ -68,8 +68,6 @@
         <label for="city-choice">Select a city: </label>
         <select name="city-choice" id="city-choice">
             <option value="ALL">Select a city</option>
-            <!-- <option value="Austin">Austin</option>
-          <option value="Baker">Baker</option> -->

```

Comment:
I imagine this was to be able to experiment with options in the toggle. But there's no node. Using `npm start` will show all the cities as options.